gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Fri Sep 30 05:10:25 BST 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
```

Comment:
This could be updated to 7.4.1, but there aren't many important features in 7.4.x relative to 7.3.x.

Reply:
I left it as 7.3.3 for parity with what gdx-setup produces. I'd be willing to change it to 7.4 in both places if someone can promise it won't double the time it takes for this to be merged.

Reply:
7.4.1 is a pretty minor set of changes; we should probably wait until 7.5 or whatever Gradle supports Java 18.